water_mark/download_water_data.pyz

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+'''Downloader for the Global Surface Water Data of the Copernicus Programme:
```

Comment:
Because we switched to using the vv and vh histogram we might not need to add this to the main repo.